js/classes/entities/midground/enemies/slime.js

```diff
@@ -117,9 +117,13 @@ class Slime {
     handleDeath() {
         this.action = "dying";
 
-        if (Math.random() < 0.3) {
-            const pos = Vector.add(this.base.getCenter(), new Vector(0, -20));
-            GAME.addEntity(new FoodDrop(pos, AmmoDrop.SLIMEBALL));
+        const rand = Math.random();
+        if (rand < 0.3) {
+            const pos = Vector.add(this.base.getCenter(), new Vector(0, -40));
+            GAME.addEntity(new AmmoDrop(pos, AmmoDrop.SLIMEBALL));
```

Comment:
Nice! The reason I wasn't seeing many drops may have had to do with the bug but this is good too.